.github/workflows/update_data.yml

```diff
@@ -8,6 +8,9 @@ on:
 
 jobs:
   update:
+    strategy:
+      matrix: 
+        orgs: ["DSACMS","Enterprise-CMCS","CMS-Enterprise","CMSgov"]
     permissions: write-all
```

Comment:
is this `write-all` for just the metrics repo, or for all the `orgs`?

Reply:
for just the metrics repo